featuretools/tests/entityset_tests/test_es.py

```diff
@@ -25,7 +25,7 @@
     SubRegionCode,
 )
 
-import featuretools as ft
+from featuretools import Relationship, demo
```

Comment:
I think we should change `demo` here to do a `from featuretools.demo import load_retail` to stick with this theme - import the function directly, not the entire demo module.